primefaces/src/main/resources/META-INF/resources/primefaces/validation/validation.common.js

```diff
@@ -511,6 +550,36 @@ if (window.PrimeFaces) {
             }
 
             return valid;
+        },
+
+
+        /**
+         * __NOTE__: This is an internal method and should only be used by PrimeFaces itself.
+         *
+         * Bind to Ajax-Complete-events to update CSV-state after an Ajax-call may have changed state.
+         * @internal
+         */
+        bindAjaxComplete: function() {
+            if (this.ajaxCompleteBound) return;
+
+            var doc = $(document),
+                $this = this;
+
+            doc.on('pfAjaxComplete', function(e, xhr, settings, args) {
```

Comment:
```suggestion
            $(document).on('pfAjaxComplete', function(e, xhr, settings, args) {
```

No need to store `doc` variable as its not used anywhere else?

Reply:
good catch - thanks!